pkg/apis/networking/types.go

```diff
@@ -490,7 +490,7 @@ type IngressServiceBackend struct {
 
 // ServiceBackendPort is the service port being referenced.
 type ServiceBackendPort struct {
-	// Name is the name of the port on the Service.
+	// Name is the name of the port on the Service. This must be an IANA_SVC_NAME (following RFC6335).
```

Comment:
nit, it looks better if we add it in the next line

Reply:
done